libs/model/src/community/GetCommunities.query.ts

```diff
@@ -109,7 +117,7 @@ export function GetCommunities(): Query<typeof schemas.GetCommunities> {
                             : ''
                         }
                         ${
-                          filtering_tags
+                          filtering_tags && loose_filter !== 'tag_ids'
```

Comment:
What is the purpose of `loose_filter` here?

Reply:
I couldn't think of a better name for this tbh, basically, its for relevance sort of matching tag_ids, results with matching tags ids will be displayed first, and then those who don't have any tag or non-matching tag ids

Reply:
I updated the name to `relevance_by` and added a comment on what would happen if its present.